checkov/sast/engines/prisma_engine.py

```diff
@@ -301,6 +301,16 @@ def create_prisma_report(self, data: Dict[str, Any]) -> PrismaReport:
             data["imports"] = {}
         if not data.get("reachability_report"):
             data["reachability_report"] = {}
+
+        remove_list = []
```

Comment:
can we extract all this new code to a function ?

Reply:
done